openmetadata-ui/src/main/resources/ui/src/components/DataAssets/CommonWidgets/CommonWidgets.tsx

```diff
@@ -139,10 +145,36 @@ export const CommonWidgets = ({
     };
   }, [updatedData, updatedData?.tags]);
 
-  const { columns = [], charts = [] } = data as unknown as {
-    columns: Array<Column>;
-    charts: Array<Chart>;
-  };
+  const isDescriptionExpanded = useMemo(() => {
+    switch (entityType) {
+      case EntityType.TABLE:
+        return isEmpty((data as unknown as Table).columns);
+      case EntityType.DASHBOARD:
+        return isEmpty((data as unknown as Dashboard).charts);
+      case EntityType.DASHBOARD_DATA_MODEL:
+        return isEmpty((data as unknown as DashboardDataModel).columns);
+      case EntityType.MLMODEL:
+        return isEmpty((data as unknown as Mlmodel).mlFeatures);
+      case EntityType.PIPELINE:
+        return isEmpty((data as unknown as Pipeline).tasks);
```

Comment:
Just wanted to check purpose of this? can you please provide some details here? or add comments that will be great.